lib/ace/ext/papeeria/katex-previewer.coffee

```diff
@@ -252,14 +254,28 @@ define((require, exports, module) ->
       curIndex = session.doc.positionToIndex({ row, column }, startRow)
       curRow = row
       curColumn = column
+      curContexts = getContexts(session, curRow, curColumn)
+      correct = true
+      reason = null
 
       while true
+        if correct and curContexts.includes(ERROR_CONTEXT)
+          correct = false
+          reason = EquationRangeHandler.WHITESPACE_LINE_ERROR_CODE
```

Comment:
Does it matter if we have empty line or whitespace line? I suggest returning (or maybe showing to the user) the same error: "Whitespace lines are not allowed in equations"

Reply:
Okay, will fix.

Reply:
Actually, we can just change error texts in i18n, we don't need to change any logic. And this is in papeeria repository, I'll add an issue over there.